platform/server/elasticsearch/api.ts

```diff
@@ -23,11 +24,25 @@ export function registerElasticsearchRoutes(
         })
       }
     },
-    async (elasticsearch, req) => {
+    async (req, res) => {
       // WOHO! Both of these are typed!
       log.info(`field param: ${req.params.field}`);
       log.info(`query param: ${req.query.key}`);
 
+      // NB: All right, so at this point when we're
+      // exposing the es API, shouldn't we have an
+      // instance of ElasticsearchService already
+      // from kibana-core? I'm a bit confused.
```

Comment:
Yep, you just need to inject it in `platform/server/elasticsearch/index.ts`, e.g.

```
return registerElasticsearchRoutes(router, this.logger, this.service);
```

Reply:
Done. makes sense!